source/installation/system-requirements.txt

```diff
@@ -0,0 +1,55 @@
+.. _rm-system-requirements:
+
+===================
+System Requirements
+===================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The following page outlines the system requirements for using Relational 
+Migrator. For additional details on where to run Relation Migrator, see 
+:ref:`rm-colocation`.
+
+Hardware
+--------
+
+Computer (can be a local desktop/labtop or remote server), 
+Virtual Machine or Docker Container with at least 8GB of RAM 
+(16GB or more recommended).
+
+Network
+-------
+
+Deploy Relational Migrator to a location close to the destination 
+databases to maximise bandwidth and minimise latency. For additional 
+details, see :ref:`rm-colocation`.
+
+Operating System
+----------------
+
```

Comment:
Consider introducing this list with something like:

"You can run Relational Migrator on one of the following operating systems: "